mono/mini/debugger-agent.c

```diff
@@ -11456,7 +11456,14 @@ string_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
 		} else {
 			MonoError error;
 			s = mono_string_to_utf8_checked (str, &error);
-			mono_error_assert_ok (&error);
+			if (!mono_error_ok(&error))
```

Comment:
Formatting doesn't match mono's needs the spaces after function call, and same line curlys